src/commcare_cloud/ansible/roles/kafka/templates/server.properties.j2

```diff
@@ -117,3 +120,9 @@ zookeeper.connect={% for host in groups['zookeeper'] %}{{ host }}:{{ zookeeper_c
 
 # Timeout in ms for connecting to zookeeper
 zookeeper.connection.timeout.ms=1000000
+{% if kafka_version == "2.4.1" %}
+inter.broker.protocol.version=0.8.2.2
```

Comment:
There are 2 options here.
### 1. make the config completely configurable:

```jinja
{% for config, value in kafka_extra_config.items() %}
{{config}}={{value}}
{% endfor %}
```
Then in the config file you can have:
```yaml
kafka_extra_config:
    inter.broker.protocol.version: 0.8.2.2
    log.message.format.version: 0.8.2.2
    etc
```

### 2. The other option is to have a separate var for each:
```jinja
inter.broker.protocol.version={{kafka_inter_broker_protocol_version}}
log.message.format.version={{kafka_log_message_format_version}}
```